constants/__init__.py

```diff
@@ -90,6 +101,58 @@
         max_bytes=15 * 1024 * 1024 * 1024,
     ),
 }
+MODEL_CONSTRAINTS_BY_COMPETITION_ID: Dict[CompetitionId, ModelConstraints] = {
+    CompetitionId.SN9_MODEL: ModelConstraints(
+        max_model_parameter_size=6_900_000_000,
+        sequence_length=4096,
+        allowed_architectures=[
+            MistralForCausalLM,
+            LlamaForCausalLM,
+            BartForCausalLM,
+            FalconForCausalLM,
+            GPTNeoXForCausalLM,
+            PhiForCausalLM,
+            GemmaForCausalLM,
+        ],
+        tokenizer="Xenova/gpt-4",
+        kwargs={
+            "torch_dtype": torch.bfloat16,
+        },
+        eval_block_delay=1200,  # ~4 hours.
```

Comment:
For consideration: Should we use a larger delay and use more samples? I'm a little concerned that 100 samples may not give a stable eval across models.

Reply:
Bumped to 200 samples. Still have some safety margin at that number in 4 hours.